toolchains/nodejs/tests/BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+package(default_testonly = 1)
+
+nodejs_binary(
```

Comment:
not sure but how is this testing any nix toolchain? Shouldn't we add some nixpkgs dependency here?

Reply:
as far as i understand the test-infrastructure here, the tests are executed in the context of the `WORKSPACE` of the toolchain, which includes the `nixpkgs_go_configure`

Reply:
That is correct.